static/src/stylesheets/module/journalism/_audio-flagship-tag-page-container.scss

```diff
@@ -95,24 +95,62 @@
         }
     }
 
-    /* ---------------- CONTRIBUTIONS ------------------- */
+    /* ---------------- CONTRIBUTIONS BANNER ------------------- */
 
     .flagship-audio__contributions-banner {
-        background-color: #333333;
-        color: $brightness-100;
+        background-color: #f0dbbc;
+        padding-bottom: 0;
 
-        .fc-container__body {
+        .flagship-audio__contributions-banner-holder {
+            display: flex;
+            flex-direction: row;
 
             h2 {
                 font-size: 26px;
                 line-height: 28px;
+                color: #ff6a56;
             }
             p {
                 font-size: 18px;
                 line-height: 20px;
             }
             a {
                 background-color: #121212;
+                max-width: 170px;
+            }
+            .flagship-audio__contributions-banner-text {
+                display: flex;
+                flex-direction: column;
+                flex: 0 0 300px;
+                z-index: 10;
+            }
+            .flagship-audio__contributions-banner-image {
+                flex: 1 1 auto;
+                display: flex;
+                justify-content: center;
+                z-index: 5;
+                svg {
+                    justify-content: center;
+                    position: absolute;
+                    bottom: 0;
+                    height: auto;
+                    width: auto;
+                    @include mq($until: phablet) {
+                        height: 64%;
```

Comment:
Interesting use of percentages.

Reply:
ahaha. Yeah. they worked. Is it better to use pixels?

Reply:
I'll make them both 60% to be more regular